backend/corpora/dataset_processing/download.py

```diff
@@ -51,6 +51,7 @@ def downloader(url: str, local_path: str, tracker: ProgressTracker, chunk_size:
         with requests.get(url, stream=True) as resp:
             resp.raise_for_status()
             with open(local_path, "wb") as fp:
+                logger.info("Starting download.")
```

Comment:
Is this configured to include the dataset id, batch job and dropbox url in the logging messages?

Reply:
no, but all of this is only visible if you know the batch job. Should we make the batch job ID visible to the users. It could make it easier for us to find their job when it fails.

Reply:
yea I think that would be helpful